monitoring/grafana/dashboards/jsonnet/grafana_dashboards.jsonnet

```
@@ -271,6 +271,12 @@ local addStyle(alias, colorMode, colors, dateFormat, decimals, mappingType, patt
     .addTemplate(
        addTemplateSchema('rgw_servers', '$datasource', 'label_values(ceph_rgw_req, ceph_daemon)', 1, true, 1, '', '')
     )
+    .addTemplate(
+       addTemplateSchema('code', '$datasource', 'label_values(haproxy_server_http_responses_total{instance=~\"$ingress_service\"}, code)', 1, true, 1, 'HTTP Code', '')
```

jsonnet doesn't need escaping `\"`:

```suggestion
       addTemplateSchema('code', '$datasource', 'label_values(haproxy_server_http_responses_total{instance=~"$ingress_service"}, code)', 1, true, 1, 'HTTP Code', '')
```